src/Vlingo.Common.Tests/BasicCompletesTest.cs

```diff
@@ -107,7 +107,7 @@ public void TestTimeoutBeforeOutcome()
             }));
             thread.Start();
 
-            completes.Await();
+            completes.Await<int>();
```

Comment:
I guess it's because .NET handles the generics differently. So we need a closed type in the end.